lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -148,3 +149,13 @@ def test_determinism_all():
 def test_determinism_all_tex():
     """Test for reproducible PS/tex output"""
     _determinism_check(format="ps", usetex=True)
+
+
+@image_comparison(baseline_images=['canonical'], extensions=["ps"])
```

Comment:
Just for my knowledge, is there a reason to use single quotes for 'canonical' but double quotes for "ps" here?

Reply:
*waves his hand* repeeat after me: "This typo never existed"